testlist/xtensa.dat

```diff
@@ -0,0 +1 @@
+/xtensa
```

Comment:
Let's move /xtensa into mips-riscv-x86.dat to save one job build in github action CI.

Reply:
I have created a copy of it as `mips-riscv-v86-xtensa.dat` but I did not delete the old `mips-riscv-x86.dat` file because doing so would break the current CI builds.  One this is all updated we can removed the unused one.

Reply:
Yes, let's remove it later.